lib/model.js

```diff
@@ -1337,7 +1337,9 @@ module.exports = (function() {
         // Filter dataValues by options.fields
         var values = {};
         options.fields.forEach(function(field) {
-          values[field] = instance.dataValues[field];
```

Comment:
I'll check that. I played around with this quite a bit, since I was not really happy with that solution (I came up with this after around three minutes and tried other stuff for at least an hour...).
I tried not to use fields all the time (that's not necessary at all) but couldn't find a good working solution.